EIPS/eip-5851.md

```diff
@@ -0,0 +1,256 @@
+
+The motivation behind this proposal is to create a standard for verifier and issuer smart contracts to communicate with each other in a more efficient way. This will reduce the cost of KYC processes, and provide the possibility for on-chain KYC checks.  By creating a standard for communication between verifiers and issuers, it will create an ecosystem in which users can be sure their data is secure and private. This will ultimately lead to more efficient KYC processes and help create a more trustful environment for users. It will also help to ensure that all verifier and issuer smart contracts are up-to-date with the most recent KYC regulations.
+
+## Specification
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+### Definitions
+
+- ZKP: stands for Zero-knowledge Proof which means that a prover can convince a verifier that an assertion is correct without providing any useful information to the verifier.
+
+- SBT: Known as Soulbound tokens, these are non-fungible and non-transferrable tokens that is used for defining the identity of the users.
+
+- SBT Certificates: SBT that represent the ownerships of ID signatures corresponding to the claims defined in `function standardClaim()`.
+
+- Verifiable credentials (VC): A Verifiable Credential is a collection of claims made by an issuer. These are temper evident credentials that allow the holders to prove that they posses certain characteristics (for example, passport verification, constraints like value of tokens in your wallet, etc) as demanded by the verifier entity.
+
+- Claims: Decentralised Identity(DID) Claims are a set of assertions that the DID Holder must fulfill to be verified.
+
+- Holder: The holder is the entity that stores the claim, such as a digital identity provider or a DID registry. The holder is responsible for validating the claim and providing verifiable evidence of the claim.
+
+- Claimer: A claimer is the party making a claim, such as in an identity verification process.
+
+- Issuer: An entity creates a verifiable credential from claims about one or more subjects to a holder. Example issuers include governments, corporations, non-profit organizations, trade associations, and individuals.
+
+- Verifier: A verifier in DID is an entity that validates data provided by an issuer of verifiable credentials, determining its accuracy, origin, currency and trustworthiness.
+
+
+
+### Metadata Standard
+
+Claims MUST be stored in the following structures:
```

Comment:
Like I said in my previous review, EIPs shouldn't make requirements about the implementation:

```suggestion
Claims MUST be exposed in the following structures:
```